submissions/GeometricalMLforNeuroscience.md

```diff
@@ -0,0 +1,215 @@
+ 
+We begin by providing an overview of how topological data analysis, a new field in machine learning, enabled breakthrough in system neuroscience.
+
+
+# Topology of representations: understanding that neuronal data is very structured.
+
+A set of points may be equipped with a topology, a mathematical scaffold that allows us to describe property of this set, as its continuity and describe a sense of locality between the points. In some cases, the set might be locally similar to a Euclidean space, our familiar finite-dimensional space described by a set of orthogonal basic elements. We might then see the set of points as a surface, a manifold. If the set of points is initially described through a common basis, the manifold is then embedded into a higher dimensional space. The goal of Topological data analysis is to start from this description of the set of points in a high dimensional space and discover the robust topological structures, as manifolds, on which the set is confined. This fast-expanding field has provided powerful algorithm for these purposes [1]. Among them, a notoriously helpful analysis is persistent homology. A ball, a simple mathematical object introduced once a topology is given to a set of points, is progressively grown around each of the data point. Intersecting balls link point together into simplexes. Holes formed in these simplexes are counted and their persistence: the range of radius for which they are present characterize their importance in describing the topology of the data. These holes are sufficient to provide a good understanding of the topology, distinguishing for example ring from torus.
+ 
+
+Persistent homology was therefore used to question if a population of neurons really represent an external variable. The head direction of the mice can be decoded in its brain from the activity of excitatory cells, notably in the Post-Subiculum and Antero-dorsal nuclei. Using recordings of the activity of such neurons [2] generated a set of population vectors, where each axis represented the mean firing rate of a neuron over short binned temporal windows. Using persistent homology, they proved that these vectors were confined on a one-dimensional manifold, which was effectively a ring. Remarkably, persistent homology could also be applied during sleep, where the animal does not move its head. Again, the set of vectors were restricted to a ring, proving that the dynamical trajectory of these ensembles of neurons was largely independent of the brain state, highlighting that its ring-like organization might be helpful for computations during wake or sleep.
```

Comment:
Some high level explanation about what persistent homology is would be great here.

Reply:
I will try to expand a bit on that.